### DISCUSSION TITLE
Does Whisper come with no cost for enterprise usage?

### DISCUSSION THREAD

**Discussion by lalithhakari** [openai/whisper#1663] [Q&A]
I've had a fantastic experience testing Whisper locally; it didn't require me to provide any API Key or similar credentials for small and medium models within my local environment. If I decide to integrate it into our production process for an enterprise-level application, could you please provide information on the pricing for its usage?

**Comment by ryanheise** [comment]
This question is answered by the README (i.e. the front page of this repository). Scroll down to the bottom to read:

> ## License
> 
> Whisper's code and model weights are released under the MIT License. See [LICENSE](https://github.com/openai/whisper/blob/main/LICENSE) for further details.

The model weights are what get downloaded when you select the small and medium models, for example. And Whisper's code is the actual Whisper program that performs the transcription using those model weights.

Both are released under the MIT License. If you click and read that license, you can see the price (or the lack thereof) and the conditions of use. It's a very short license and an easy read. You probably won't need a lawyer to help you interpret it (but since I'm not a lawyer, I will refrain from interpreting it for you, you can go ahead and read it yourself.)

**Comment by glangford** [comment]
In addition to @ryanheise 's excellent answer, note that there are also hosted implementations of whisper, forks of this repo, as well as this open source version. See for example the links below. So for enterprise use, you have the option to choose one of the various implementations.

Announcing the Preview of OpenAI Whisper in Azure OpenAI service and Azure AI Speech
https://techcommunity.microsoft.com/t5/azure-ai-services-blog/announcing-the-preview-of-openai-whisper-in-azure-openai-service/ba-p/3928388

Faster Whisper transcription with CTranslate2
https://github.com/guillaumekln/faster-whisper

whisper.cpp
https://github.com/ggerganov/whisper.cpp
